File: selectionsort.java
import java.util.*;
public class rogh{
    public static void prsum(int n,int arr[]){
        int temp=0,minpos;
     for(int i=0;i<n;i++){
       minpos=i;
        for(int j=i+1;j<n;j++){
            if(arr[i]>arr[j]){
                minpos=j;
            }
        }
         temp=arr[minpos];
        arr[minpos]=arr[i];
        arr[i]=temp;

     }
    

     for(int i=0;i<n;i++){
        System.out.print(arr[i]+" ");
        }
     

     }
    
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        System.out.println("Enter elements:\n");
        
        int arr[]=new int[n];
        for(int i=0;i<n;i++){
        arr[i]=sc.nextInt();
        }

        for(int i=0;i<n;i++){
            System.out.print(arr[i]);
            }
            System.out.print("\n");
            

            prsum(n,arr);
            
    }}

File: selectionsort.c
#include <stdio.h>
#include <stdlib.h>

void printarr(int arr[], int size)
{
    for (int i = 0; i < size; i++)
    {
        printf("%d ", arr[i]);
    }
    return;
}
void selectionsort(int *arr, int size)
{
    int min;
    for (int i = 0; i < size - 1; i++)
    {
        min = i;
        for (int j = i + 1; j < size; j++)
        {
            if (arr[j] < arr[min])
                min = j;
        }
        if (min != i)
        {
            int temp = arr[i];
            arr[i] = arr[min];
            arr[min] = temp;
        }
    }

    return;
}
int main()
{
    printf("Enter Size :\n");
    int size;
    scanf("%d", &size);
    int arr[size];
    printf("Enter Elements:\n");
    for (int i = 0; i < size; i++)
    {
        scanf("%d", &arr[i]);
    }

    selectionsort(arr, size);
    printarr(arr, size);
}